FieldOpt/Reservoir/grid/eclgrid.cpp

```diff
@@ -66,58 +75,99 @@ namespace Reservoir {
         {
             Dims dims;
             if (type_ == GridSourceType::ECLIPSE) {
-                ERTWrapper::ECLGrid::ECLGridReader::Dims eclDims = ecl_grid_reader_->Dimensions();
-                dims.nx = eclDims.nx;
-                dims.ny = eclDims.ny;
-                dims.nz = eclDims.nz;
+                ERTWrapper::ECLGrid::ECLGridReader::Dims
```

Comment:
The indentation here too: auto indent will change it. Also, the `dims` object is not the one created above, which is confusing. To short down the line with the intializer, `auto` could have been used.